src/flint/FlintTypes.jl

```diff
@@ -42,7 +44,7 @@ integer_ring() = ZZRing()
 
 @doc zz_ring_doc
 mutable struct ZZRingElem <: RingElem
-  d::Int
+  d::Flint.fmpz
```

Comment:
```suggestion
  data::Flint.fmpz
```
consistency with the other structs